linux-ppc64le-anvil/Dockerfile

```diff
@@ -0,0 +1,88 @@
+FROM ppc64le/centos:7
+
+MAINTAINER conda-forge <conda-forge@googlegroups.com>
+
+
+# Set an encoding to make things work smoothly.
+ENV LANG en_US.UTF-8
+
+# Add a timestamp for the build. Also, bust the cache.
+ADD http://tycho.usno.navy.mil/timer.html /opt/docker/etc/timestamp
+
+# Resolves a nasty NOKEY warning that appears when using yum.
+RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+
+# Install basic requirements.
+RUN yum update -y && \
+    yum install -y \
+                   file \
+                   bzip2 \
+                   make \
+                   patch \
+                   sudo \
+                   tar \
+                   which \
+                   libXau \
+                   libxcb \
+                   libXdmcp \
+                   libXext-devel \
+                   libXrender-devel \
+                   libSM-devel \
+                   libX11-devel \
+                   libXt \
+                   mesa-libGL-devel \
+                   mesa-libGLU \
+                   openssl \
+                   openssh-clients \
+                   rsync \
+                   util-linux \
+                   wget \
+                   xorg-x11-server-Xvfb && \
+    yum clean all
+
+# give sudo permission for conda user to run yum (user creation is postponed
+# to the entrypoint, so we can create a user with the same id as the host)
+RUN echo 'conda ALL=NOPASSWD: /usr/bin/yum' >> /etc/sudoers
+
+# Install the latest Miniconda with Python 3 and update everything.
+RUN curl -s -L https://repo.continuum.io/miniconda/Miniconda3-4.3.27-Linux-ppc64le.sh > miniconda.sh && \
+    openssl md5 miniconda.sh | grep c51bc5fb00da12b487959b5adf0133ab && \
+    bash miniconda.sh -b -p /opt/conda && \
+    rm miniconda.sh && \
+    touch /opt/conda/conda-meta/pinned && \
+    export PATH=/opt/conda/bin:$PATH && \
+    conda config --set show_channel_urls True && \
+    conda config --add channels conda-forge && \
+    conda config --add channels furmanek && \
+    conda update --all --yes && \
+    conda clean -tipy
+
+# Install conda build and deployment tools.
+RUN export PATH="/opt/conda/bin:${PATH}" && \
+    conda install --yes --quiet conda-build anaconda-client jinja2 setuptools && \
+    conda install --yes git && \
+    conda clean -tipsy
+
+# Install docker tools
+RUN export PATH="/opt/conda/bin:${PATH}" && \
+    conda install --yes su-exec && \
+    export CONDA_SUEXEC_INFO=( `conda list su-exec | grep su-exec` ) && \
+    echo "su-exec ${CONDA_SUEXEC_INFO[1]}" >> /opt/conda/conda-meta/pinned && \
+    conda clean -tipsy
+
+# Add tini from git
+ENV TINI_VERSION v0.18.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-ppc64el /tini-ppc64el
+RUN chmod +x /tini-ppc64el
```

Comment:
We also build `tini`. If you are able to build a copy of this for PPC using our recipe, we could upload it as well and use the `conda` package for it as well.

Long term we could build `tini` for PPC in conda-forge and just dogfood it here.

ref: https://github.com/conda-forge/tini-feedstock

Reply:
Yup. That's a good idea. The recipe for tini will likely work. I'll give it a shot.

Reply:
While you are building packages, please build `conda-forge-ci-setup` as well.

Reply:
Oh, I remember I did try that. The `tini` package depends on `toolchain`, which is one of the ones we'll want to spend some time on I think.

Reply:
`toolchain` built fine. I may need a small change to the `tini` recipe though.
I'll check out the ci-setup one too.

Reply:
So `toolchain` in `conda-forge` doesn't do anything too exciting. It just [copies]( https://github.com/conda-forge/toolchain-feedstock/blob/bf0bfa1f06f6592eb3e6ef8989339a005cfd93ef/recipe/install-toolchain.sh#L8 ) this [activation script]( https://github.com/conda-forge/toolchain-feedstock/blob/bf0bfa1f06f6592eb3e6ef8989339a005cfd93ef/recipe/activate.sh ) and runs it when the package gets installed and the environment activated.

That said, `toolchain` is legacy tooling and I think we want to use the compilers from `defaults` for PPC, right? In which case we might want to avoid using the `toolchain` package. Have merged @isuruf's PR ( https://github.com/conda-forge/tini-feedstock/pull/4 ), which should make using `defaults` compilers easier.

Please let us know if you run into any issues with this. Happy to help. :)

Reply:
`tini` built fine now.  I have both `tini` and `conda-forge-cu-setup` at:
https://anaconda.org/furmanek/repo

Reply:
I'll do it tonight, if nobody beats me to it.

Reply:
Done.

Reply:
OK, I see it here:
https://anaconda.org/conda-forge/tini/files

but I get:

```
PackagesNotFoundError: The following packages are not available from current channels:

  - tini

Current channels:

  - https://conda.anaconda.org/conda-forge/linux-ppc64le
  - https://conda.anaconda.org/conda-forge/noarch
  - https://repo.anaconda.com/pkgs/main/linux-ppc64le
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/free/linux-ppc64le
  - https://repo.anaconda.com/pkgs/free/noarch
  - https://repo.anaconda.com/pkgs/r/linux-ppc64le
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://repo.anaconda.com/pkgs/pro/linux-ppc64le
  - https://repo.anaconda.com/pkgs/pro/noarch
```
What do you think?

Reply:
It was in a different label. Fixed now